### DISCUSSION TITLE
Async on initial load, then sync read & write

### DISCUSSION THREAD

**Discussion by wong2** [pmndrs/jotai#2244] [Q&A]
Hi, I need to implement this pattern: load the initial value for an atom asynchronously (with suspense mode). Afterward, writes should just modify the loaded value in memory, and read should use this memory value.
I've read all the documents but could not find an example related to this.

**Comment by dai-shi** [comment]
See #2228, and you may also need to combine with `atomWithDefault`.

**Reply by wong2** [reply]
Thanks, I've read the discussion, but I can't understand what `unwrap` does event after reading the docs. Can you help get me into it?

**Reply by dai-shi** [reply]
https://github.com/pmndrs/jotai/blob/f5691b5a6a68e5b0f5ae05390a45bff309900ea3/docs/utilities/async.mdx#L84

Can you try running the example with https://csb.jotai.org? (If it works, please open a PR to add the example in docs.)
